test/GithubApi.Put.test.js

```diff
@@ -0,0 +1,45 @@
+require('dotenv').config();
+
+const { StatusCodes } = require('http-status-codes');
+const { expect } = require('chai');
+const axios = require('axios');
+
+const urlBase = 'https://api.github.com';
+const githubUserName = 'aperdomob';
+
+describe('Consume PUT Method', () => {
+  let followResponse;
+  let followList;
+  let username;
+
+  before(async () => {
+    followResponse = await axios.put(`${urlBase}/user/following/${githubUserName}`, {}, {
+      headers: {
+        Authorization: `token ${process.env.ACCESS_TOKEN}`
+      }
+    });
+
+    followList = await axios.get(`${urlBase}/user/following`, {
+      headers: {
+        Authorization: `token ${process.env.ACCESS_TOKEN}`
+      }
+    });
+
+    username = followList.data.find((element) => element.login === 'aperdomob');
+  });
+
+  it('Follow User with PUT Method', async () => {
+    expect(followResponse.status).to.equal(StatusCodes.NO_CONTENT);
+    expect(followResponse.data).to.equal('');
+  });
+
+  it('Check following list to confirm', async () => {
+    expect(followList.status).to.equal(StatusCodes.OK);
+    expect(username.login).to.equal('aperdomob');
+  });
+
+  it('Check idempotence of the PUT Method', async () => {
+    expect(followResponse.status).to.equal(StatusCodes.NO_CONTENT);
+    expect(followResponse.data).to.equal('');
+  });
```

Comment:
Please add another test to verify that you are still following the user 'aperdomob'